rust-toolchain.toml

```diff
@@ -0,0 +1,13 @@
+[toolchain]
+# rustup has no subcommand (yet) for installing the toolchain in rust-toolchain.toml:
+# https://github.com/rust-lang/rustup/issues/2686
+# The auto-installation behaviour in rustup will likely be removed:
+# https://github.com/rust-lang/rustup/issues/1397
+channel = "1.75.0"
```

Comment:
Parity uses the latest stable, so upgraded our toolchain to latest stable. We could also just change this to "stable" in the future.